app/services/catalog/notify.rb

```diff
@@ -0,0 +1,37 @@
+module Catalog
+  class Notify
+    ACCEPTABLE_NOTIFICATION_CLASSES = %w[OrderItem ApprovalRequest].freeze
+    EVENT_REQUEST_FINISHED = "request_finished".freeze
+
+    attr_reader :notification_object
+
+    def initialize(klass, id, payload)
```

Comment:
@eclarizio One small question. Should I modify the Minion to pass the klass as `approval_request, order_item` snake case? That would fit better with standard parameter naming.

Reply:
@syncrou Yeah, that's fine, I think the specs treat it like that, plus we're converting to `camelcase` anyway before we `constantize`.